UnityProject/Assets/Scripts/NPC/AI/MobExplore.cs

```diff
@@ -75,6 +94,80 @@ public void OnSpawnServer(SpawnInfo info)
 			eatFoodSound = CommonSounds.Instance.EatFood;
 		}
 
+		//https://www.geeksforgeeks.org/bresenhams-circle-drawing-algorithm/
+		// Function for circle-generation
+		// using Bresenham's algorithm
+		void circleBres(int xc, int yc, int r)
```

Comment:
in these 3 methods use the explicit access level instead of implicitly being private. Also use PascalCase for methods